packages/shared/kebab/kebab.tsx

```diff
@@ -30,20 +30,20 @@ type KebabProps = {
   isDisabled?: boolean;
 };
 
-const defaultKebabItems = (t: TFunction) => ({
+const defaultKebabItems = (t: TFunction, resourceLabel: string) => ({
   [ModalKeys.EDIT_LABELS]: (
     <DropdownItem key={ModalKeys.EDIT_LABELS} id={ModalKeys.EDIT_LABELS}>
-      {t('plugin__odf-console~Edit Labels')}
+      {t('plugin__odf-console~Edit labels')}
```

Comment:
```suggestion
      {t('Edit labels')}
```

Please remove "plugin__odf-console~", it will create problem for MCO plugin.

Reply:
ack